File: benchmarks/suite/fstr_format.go
package main

import "fmt"

func main() {
	var sum int64
	const N = 500_000
	for i := 0; i < N; i++ {
		s := fmt.Sprintf("item=%d", i)
		sum += int64(len(s))
	}
	fmt.Println(sum)
}

File: benchmarks/suite/fstr_format.cpp
#include <cstdio>
#include <string>
int main() {
    long long sum = 0;
    const int N = 500000;
    for (int i = 0; i < N; i++) {
        std::string s = std::string("item=") + std::to_string(i);
        sum += (long long)s.size();
    }
    printf("%lld\n", sum);
}
